### DISCUSSION TITLE
Can I use btop to set nice values (change priority)?

### DISCUSSION THREAD

**Discussion by Felipe-noob** [aristocratos/btop#32] [Q&A]
I'd like to know if there's already a feature to change priority (like in KsysGuard) or simple nice values (like in htop). If it's not yet implemented, I'll put this as a suggestion.

**Comment by LinuxOpa** [comment]
this is a menu item that I am also missing.

**Comment by Zaczero** [comment]
The problem with GitHub Discussions is that barely anyone monitors them, yet so many people push for them.

**Comment by phanirithvij** [comment]
I think this is implemented, when you hit enter on a process, there is [N]ice available.
